app/components/LargeHeader.js

```diff
@@ -19,7 +26,7 @@ function LargeHeader() {
         <div className="search-container form-row" role="search">
           <input type="text" className="search-field" placeholder="I'm looking for..." name="srch-term" id="srch-term" />
           <input type="text" className="location-field" placeholder="Near me" name="srch-location" id="srch-location" />
-          <button id="largeheader_searchbutton" className="button search" type="submit"><i className="material-icons">search</i> <span>Search</span></button>
+          <button onClick={submitSearch} id="largeheader_searchbutton" className="button search" type="submit"><i className="material-icons">search</i> <span>Search</span></button>
```

Comment:
I suspect we need to more than onClick here, is there a best practice on handling any sort of input field submit? (enter key or click on button?)